packages/server/src/schema/context.ts

```diff
@@ -15,7 +15,7 @@ export class SimulationContext {
     let simulation = simulations.slice(id);
     simulation.set({ id, status: 'new', simulators, scenarios: {}, store: {} });
 
-    return scope.spawn(simulation.once(settled));
```

Comment:
I'm kind of sad to see `once` go.  Was the motivation to get people more used to the API?  Could `once` not call `filter(o).expect()`

It is no big deal but slightly curious

Reply:
I like the once method too. The main thing though is that it's extra API to add to atom that you need to know beyond what is provided by Stream. The thing that's nice now is that if you know stream, you don't need anything else.

What about adding `once()` upstream to `Stream`? That could be nice.

/cc @jnicklas